datumaro/plugins/yolo_detection_format/importer.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import logging as log
+import os.path as osp
+from glob import glob
+import yaml
+from collections import defaultdict, Counter
+from typing import Any, Dict, List
+from io import TextIOWrapper
+
+from datumaro.components.errors import DatasetImportError
+from datumaro.components.errors import DatasetNotFoundError
+from datumaro.components.extractor import DEFAULT_SUBSET_NAME, Importer
+from datumaro.components.format_detection import FormatDetectionConfidence, FormatDetectionContext
+from datumaro.plugins.yolo_detection_format.extractor import YoloDetectionExtractor
+from datumaro.util.os_util import extract_subset_name_from_parent
+
+from .format import YoloDetectionPath
+
+
+class YoloDetectionImporter(Importer):
+    META_FILE = YoloDetectionPath.META_FILE
```

Comment:
It is the pain. Please find a way to redefine the default config file name. We always should be able to treat it as a configuration parameter. If it isn't specified, I would recommend for looking for *.yaml files inside. If only one file is found, you can proceed. If several files with the same name are found, you need to report an error.

Algorithm and recommendations:
* please add a CLI argement to specify config file name
* if you don't have a hint from the command line, please try to find all *.yaml files
* If you find only one yaml file, it is the config
* if you find several yaml files, check that data.yaml exists. If it is the case, use it.
* Otherwise please report an error.
AR